directives/vaadin-combo-box.js

```diff
@@ -63,6 +63,7 @@ System.register(['angular2/core', 'angular2/common', 'angular2/src/facade/lang']
                     }
                 };
                 VaadinComboBox.prototype.valuechanged = function (value) {
+                    var _this = this;
```

Comment:
Can’t we use ES5 `bind()` for these? Would look a bit cleaner IMO. But I don’t really care too much :)